lib/screens/navigate/my_inventory.dart

```diff
@@ -0,0 +1,154 @@
+          builder: (BuildContext c) {
+            return Dialog(
+              shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(20)),
+              child: Container(
+                height: 450,
+                width: MediaQuery.of(context).size.width,
+                decoration: new BoxDecoration(
+                    borderRadius: BorderRadius.circular(20),
+                    color: AppColors.BACKGROUND_COLOR),
+                child: SingleChildScrollView(
+                  child: Column(children: <Widget>[
+                    Container(
+                      margin: EdgeInsets.fromLTRB(8, 16, 8, 16),
+                      child: Text(
+                        'Device Details',
+                        style: TextStyle(
+                          color: AppColors.LIGHTBLACK_COLOR,
+                          fontSize: 25,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                    ),
+                    DropDown(),
+                    InventoryItemDetails(
+                      text1: "BSY ",
+                      text2: "Chargers",
+                    ),
+                    InventoryItemDetails(
+                      text1: "id",
+                      text2: "504",
+                    ),
+                    InventoryItemDetails(
+                        text1: "Assign Date", text2: "2018-08-08"),
+                    InventoryItemDetails(
+                      text1: "Serial Number",
+                      text2: "BSYE1202000",
+                    ),
+                    InventoryItemDetails(
+                      text1: "Internal Serial Nummber ",
+                      text2: "CHA001",
+                    ),
+                    InventoryItemDetails(
+                      text1: "Deepak Mishra",
+                      text2: "No Designation Data",
+                    ),
+                    SizedBox(
+                      height: 30,
+                    ),
+                    for (int i = 0; i < 5; i++)
+                      CommentSection(
+                        name: 'Deepak Mishra',
+                        date: '6th April20, 11:55 am',
+                        comment: 'All Good',
+                      ),
+                  ]),
+                ),
+              ),
+            );
+          });
+    }
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Text('My Inventory',
+                style: TextStyle(
+                    color: Colors.white,
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold)),
+            CircleAvatar(
+                radius: 20,
+                backgroundImage: AssetImage('assets/images/person.jpg'))
+          ],
+        ),
+      ),
+      drawer: Navigation(),
+      floatingActionButton: CupertinoButton(
```

Comment:
@Shaktiexcellence where's ios condition and where is material for ios

Reply:
IOS condition is written only once in Application , in Home page to return CupertinoApp or MaterialApp